packages/kbn-optimizer/limits.yml

```diff
@@ -118,3 +118,4 @@ pageLoadAssetSize:
   reporting: 57003
   visTypeHeatmap: 25340
   screenshotting: 17017
+  expressionGauge: 94959
```

Comment:
Is this adding 100kB to the page load, or is the Lens entry being reduced as a result?

Reply:
Lens is being reduced too (lens | 1.1MB | 1015.1KB | -69.0KB) but I'll lazy load it.